x-pack/plugins/cloud/public/types.ts

```diff
@@ -225,3 +225,7 @@ export interface CloudSetup {
     orchestratorTarget?: string;
   };
 }
+
```

Comment:
I think we should add a code doc for this type. and Explicitly state that `elasticsearchUrl` can be undefined and an empty string `''` to try and help consumers know how to best handle all results.